lib/dependencies/ExportsInfoDependency.js

```diff
@@ -51,23 +51,19 @@ const getProperty = (moduleGraph, module, exportName, property, runtime) => {
 	switch (property) {
 		case "canMangle": {
 			const exportsInfo = moduleGraph.getExportsInfo(module);
-			const exportInfo = exportsInfo.getExportInfo(
-				/** @type {string} */ (exportName)
-			);
+			const exportInfo = exportsInfo.getExportInfo(exportName[0]);
```

Comment:
Intresting... Why we used `exportName` before

Reply:
I guess the TODO type is the chief culprit XD 
https://github.com/webpack/webpack/pull/18060/files#diff-f1ace2ea8415a88f3492366da7b387082adf5d366ad66b6cbac4edb0a28590f7L96

Reply:
This should be calling `getReadonlyExportInfoRecursive(exportName)`.

The other cases all descend fully down the tree, but this one currently only goes one layer deep, which makes unit test results that use this property misleading.